dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -150,30 +397,18 @@ def test_save_and_load(self):
             mock_file.seek(0)
             load_profile = dp.GraphProfiler.load("mock.pkl")
 
-        # Removed to avoid dict equality ambiguity
-
-        save_properties = save_profile.profile["continuous_distribution"]["weight"].pop(
-            "properties"
-        )
-        load_properties = load_profile.profile["continuous_distribution"]["weight"].pop(
-            "properties"
-        )
-        self.assertTrue(
-            np.array_equal(np.hstack(save_properties), np.hstack(load_properties))
-        )
-
         # Check that reports are equivalent
         save_report = save_profile.report()
         load_report = load_profile.report()
         self.assertDictEqual(save_report, load_report)
 
         # adding new data and updating profiles
-        self.graph.add_edges_from(
+        self.graph_1.add_edges_from(
```

Comment:
I'm realizing this actually might be dangerous as we are changing the graph in the `self`. If we copied it, it would be okay, but in this case this could affect pass fail depending on the order of the tests running since we are changing the graph.

Reply:
actually, do we need this line even? we could technically just send the graph in again. It will assume it is a new separate graph.